src/vs/workbench/contrib/search/browser/searchActions.ts

```diff
@@ -5,146 +5,1406 @@
+			},
+			menu: [{
+				id: MenuId.SearchContext,
+				when: Constants.FileMatchOrMatchFocusKey,
+				group: 'search_2',
+				order: 1
+			}]
+		});
+
+	}
+
+	override async run(accessor: ServicesAccessor, match: RenderableMatch | undefined): Promise<any> {
+		copyMatchCommand(accessor, match);
+	}
+}
+
+export class CopyPathCommandAction extends Action2 {
+
+	constructor(
+	) {
+		super({
+			id: Constants.CopyPathCommandId,
+			title: {
+				value: nls.localize('copyPathLabel', "Copy Path"),
+				original: 'Copy Path'
+			},
+			category: category.value,
+			keybinding: {
+				weight: KeybindingWeight.WorkbenchContrib,
+				when: Constants.FileMatchOrFolderMatchWithResourceFocusKey,
+				primary: KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.KeyC,
+				win: {
+					primary: KeyMod.Shift | KeyMod.Alt | KeyCode.KeyC
+				},
+			},
+			menu: [{
+				id: MenuId.SearchContext,
+				when: Constants.FileMatchOrFolderMatchWithResourceFocusKey,
+				group: 'search_2',
+				order: 2
+			}]
+		});
+
+	}
+
+	override async run(accessor: ServicesAccessor, fileMatch: FileMatch | FolderMatchWithResource | undefined): Promise<any> {
+		copyPathCommand(accessor, fileMatch);
+	}
+}
+
+export class CopyAllCommandAction extends Action2 {
+
+	constructor(
+	) {
+		super({
+			id: Constants.CopyAllCommandId,
+			title: {
+				value: nls.localize('copyAllLabel', "Copy All"),
+				original: 'Copy All'
+			},
+			category: category.value,
+			menu: [{
+				id: MenuId.SearchContext,
+				when: Constants.HasSearchResults,
+				group: 'search_2',
+				order: 3
+			}]
+		});
+
+	}
+
+	override async run(accessor: ServicesAccessor): Promise<any> {
+		copyAllCommand(accessor);
+	}
+}
+
+/** ACTIONS: SYMBOL ACTIONS */
+
+export class ShowAllSymbolsAction extends Action2 {
+
+	static readonly ID = 'workbench.action.showAllSymbols';
+	static readonly LABEL = nls.localize('showTriggerActions', "Go to Symbol in Workspace...");
+	static readonly ALL_SYMBOLS_PREFIX = '#';
+
+	constructor(
+	) {
+		super({
+			id: Constants.ShowAllSymbolsActionId,
+			title: {
+				value: nls.localize('showTriggerActions', "Go to Symbol in Workspace..."),
+				original: 'Go to Symbol in Workspace...',
+				mnemonicTitle: nls.localize({ key: 'miGotoSymbolInWorkspace', comment: ['&& denotes a mnemonic'] }, "Go to Symbol in &&Workspace...")
+			},
+			f1: true,
+			keybinding: {
+				weight: KeybindingWeight.WorkbenchContrib,
+				primary: KeyMod.CtrlCmd | KeyCode.KeyT
+			},
+			menu: {
+				id: MenuId.MenubarGoMenu,
+				group: '3_global_nav',
+				order: 2
+			}
+		});
+	}
+
+	override async run(accessor: ServicesAccessor): Promise<void> {
+		accessor.get(IQuickInputService).quickAccess.show(ShowAllSymbolsAction.ALL_SYMBOLS_PREFIX);
+	}
+}
+
+export class ExecuteWorkspaceSymbolProviderAction extends Action2 {
+	constructor() {
+		super({
+			id: Constants.ExecuteWorkspaceSymbolProviderID,
+			title: {
+				value: nls.localize('ExecuteWorkspaceSymbolProviderID.label', 'Execute Workspace Symbol Provider'),
```

Comment:
This one is special, it's sort of an internal API command (that's why the id starts with `_` and probably shouldn't actually have a real title to show that it's not meant for display)

Reply:
restored how it was before in `search.contributions.ts`